File: app/src/main/java/com/glwhiteboard/MainActivity.java
package com.glwhiteboard;

import android.app.Activity;
import android.os.Bundle;

import androidx.annotation.Nullable;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(new WhiteBoardGLSurfaceView(this));
    }
}

File: app/src/main/java/com/cjztest/glDCTDenoise/MainActivity.kt
package com.cjztest.glDCTDenoise

import android.app.Activity
import android.opengl.GLSurfaceView
import android.os.Bundle

class MainActivity : Activity() {
    val s = android.opengl.GLSurfaceView(this)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val render = DCTRGBRenderer(this).apply {
//            setInputBitmap(Bitma.)
        }
        s.setEGLContextClientVersion(3)
        s.setRenderer(render)
        s.renderMode = GLSurfaceView.RENDERMODE_WHEN_DIRTY;
    }

    override fun onResume() {
        super.onResume()
        s.requestRender()
    }

}